README.md

```diff
@@ -54,6 +54,19 @@ $shipmentResponse = $client->shipments()->create(
 $label = $shipmentResponse->getLabel();
 ```
 
+### Adding ProductOptions to a Shipment
+```
+$shipment = new Shipment();
+$productOptions = [
+    ( new ProductOption() )->setOption( ProductOption::OPTION_ALLOW_NEIGHBOURS )->setValue( true ),
```

Comment:
Could you remove the white spaces between ( ), so it looks like this:
`(new ProductOption())->setOption(ProductOption::OPTION_ALLOW_NEIGHBOURS)->setValue(true),`